org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler.yml

```diff
@@ -0,0 +1,30 @@
+id: org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler
+branch: '22.08'
+build-extension: true
+runtime: org.freedesktop.LinuxAudio.BaseExtension
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-options:
+  prefix: /app/extensions/Plugins/NeuralAmpModeler
+modules:
+  - name: NeuralAmpModeler
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    post-install:
+      - "mv ${FLATPAK_DEST}/lib/lv2 ${FLATPAK_DEST}"
+      - "install -Dm644 LICENCE.md ${FLATPAK_DEST}/share/licenses/NeuralAmpModelerLV2Plugin/LICENSE"
+      - "install -Dm644 deps/NeuralAmpModelerCore/LICENSE ${FLATPAK_DEST}/share/licenses/NeuralAmpModeler/LICENSE"
+      - "install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler.metainfo.xml"
+      - "appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}"
+    cleanup:
+      - lib
+    sources:
+      - type: git
+        url: https://github.com/mikeoliphant/neural-amp-modeler-lv2
+        tag: 0.1.0
+        commit: 0aad7c32b6806c826848b21bb1d2699eb79de29b
```

Comment:
```suggestion
        url: https://github.com/mikeoliphant/neural-amp-modeler-lv2
        commit: cde3df4e84b0a577d48b49de4aa451b1b3ef0171
```

Reply:
this is the next commit that fix the submodules by using a more recent commit. At least this build.